### DISCUSSION TITLE
Make a Shavian-to-Latin dictionary?

### DISCUSSION THREAD

**Discussion by trosel** [Shavian-info/readlex#40] [Ideas]
I'm not sure if we would need a new dictionary, but the idea is, if someone searches "𐑐𐑹", it could show both "pore" and "poor". And if someone searches "𐑐𐑫𐑼", it could show just "poor".

Theoretically, in such a dictionary, we could allow for more varied Shavian spellings of words to link to the same Latin words. Because it would use Shavian as the primary column instead of Latin.

**Comment by Shavian-info** [comment]
This is already available with the searchable [Read Lexion](https://readlex.pythonanywhere.com/). I don't think I have 𐑐𐑹 as an alternative spelling for "poor", though. I do have to be selective about alternative spellings given I could add tens of thousands for months or years on end of them without ever truly expanding the Lexicon, which seems like a waste of time. But maybe the CURE lexical set deserves a bit of attention given how unstable it is.

EDIT: Also, the raw data for the ReadLex is script neutral, if you look at readLex.json (which is the file from which all other forms are derived).

**Reply by trosel** [reply]
> This is already available with the searchable [Read Lexion](https://readlex.pythonanywhere.com/).

Oh for some reason I thought that we lost that functionality in the upgrade. It's nice that the asterisk trick still works too :)

> I do have to be selective about alternative spellings given I could add tens of thousands for months or years on end of them without ever truly expanding the Lexicon

That is exactly what I am thinking of yes. Thinking ahead, imagine if you learn Shavian and you are trying to search for the word "poor". But you are searching for it how you say it "𐑐𐑹" and nothing is coming up. Ideally, the data format would allow for multiple spellings that are linked to the same "word sense". 

Maybe even tag each spelling with the major region that it occurs in (like how we have GA and RP in the readlex today). 

I'm not asking you personally to do this btw. That is the beauty of the github. Anyone can contribute as they would like to.
